official/vision/beta/projects/yolo/dataloaders/classification_vision.py

```diff
@@ -0,0 +1,118 @@
+"""Classification parser."""
+
+# Import libraries
+import tensorflow as tf
+import tensorflow_datasets as tfds
+import tensorflow_addons as tfa
+
+from official.vision.beta.dataloaders import parser
+from official.vision.beta.ops import preprocess_ops
+from official.vision.beta.ops import augment
+
+
+class Parser(parser.Parser):
+  """Parser to parse an image and its annotations into a dictionary of tensors."""
+
+  def __init__(self,
+               output_size,
+               aug_policy,
+               scale=[128, 448],
+               dtype='float32'):
+    """Initializes parameters for parsing annotations in the dataset.
```

Comment:
Can you fix Args mismatch in the docstr.

Reply:
fixed